### PR TITLE
Mitra/77874 add whatsapp to homepage

### PR DESCRIPTION
**Pull request by mitra-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — DISMISSED** [review]

File: src/components/layout/layout.tsx [hunk 1]
```diff
@@ -318,6 +319,7 @@ const Layout = ({
             )}
 
             {!no_live_chat && <LiveChat is_banner_shown={show_cookie_banner} />}
+            <WhatsApp />
```
Comment (NikitK-deriv) [inline src/components/layout/layout.tsx:322]: Seems it should be after `{FooterNav}`. Isn't it?
Is icon should hide right now?

Comment (mitra-deriv) [reply src/components/layout/layout.tsx:322]: Hi @NikitK-deriv why it should be after {FooterNav} ? 🤔 

No we won’t hide it , we showing it to Nigeria and South African clients 🙏

Comment (NikitK-deriv) [reply src/components/layout/layout.tsx:322]: Ohh okay then 👍  I passed it.

File: src/components/layout/whatsapp.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import { whatsapp_url } from 'common/constants'
+
+const StyledWhatsApp = styled.div`
+    position: fixed;
+    bottom: 9rem;
+    right: 1.6rem;
+    background-color: var(--color-white);
+    box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.05), 0 0 20px 0 rgba(0, 0, 0, 0.05);
+    -webkit-tap-highlight-color: transparent;
+    user-select: none;
```
[the remark this answers is not in the document]
Comment (mitra-deriv) [reply src/components/layout/whatsapp.tsx:16]: Not necessary really I will remove it thanks for pointing it out 🙏

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**